lib/outputTemplate.mustache

```diff
@@ -1,18 +1 @@
-<div class="reactPatternContainer" id="reactPatternContainer--{{{patternPartial}}}">
-  {{{htmlOutput}}}
-</div>
-
-<!-- pattern JSON (React props) -->
-<script data-type="patternJSON" id="patternJSON--{{{patternPartial}}}" type="application/json">
-  {{{json}}}
-</script>
-
-<!-- runtime React output -->
-<script>
-  {{{runtimeCode}}};
-
-  <!-- runtime rendering -->
-  var component = unknown.default;
-  var patternJSON = document.getElementById('patternJSON--{{{patternPartial}}}').textContent;
-  ReactDOM.render(React.createElement(component, JSON.parse(patternJSON)), document.getElementById('reactPatternContainer--{{{patternPartial}}}'));
-</script>
```

Comment:
Yep, just as discussed in gitter i'm going to revert it back in. In addition to this i'll include a flag to disable it. This allows teams to roll their own bundler & engine if they want to.